native/test/src/tests/thumbnail.rs

```diff
@@ -518,7 +518,7 @@ async fn image_attachment_can_support_thumbnail() -> Result<()> {
     assert_eq!(attachment.event_id(), attachment_id);
     assert_eq!(attachment.type_str(), "image");
 
-    let msg_content = attachment.msg_content();
+    let msg_content = attachment.msg_content().unwrap();
```

Comment:
is this safe? Better give ```.expect()``` here for tests easier debugging?

Reply:
in tests we get full tracebacks including line-numbers, `unwrap` is fine here.